ui.tests/test-module/specs/title/titleV2.authoring.spec.js

```diff
@@ -0,0 +1,138 @@
+            titleDrop = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.title.split("/").pop();
+
+        beforeEach(function () {
+            // this is done since cypress session results in 403 sometimes
+            cy.openAuthoring(pagePath);
+        });
+
+        it('check edit dialog availability of Title', function () {   
+            cy.cleanTitleTest(titleEditPath).then(() => {
+                testTitleEditDialog(titleEditPathSelector, titleDrop, false);   
+            })
+        });
+        
+    });
+
+    context('Open Sites Editor', function () {
+        const pagePath = "/content/core-components-examples/library/adaptive-form/title",
+            titleEditPath = pagePath + afConstants.RESPONSIVE_GRID_DEMO_SUFFIX + "/guideContainer/title",
+            titleEditPathSelector = "[data-path='" + titleEditPath + "']",
+            editDialogConfigurationSelector = "[data-action='CONFIGURE']",
+            titleDrop = pagePath + afConstants.RESPONSIVE_GRID_DEMO_SUFFIX + '/guideContainer/' + afConstants.components.forms.resourceType.title.split("/").pop();
+
+        beforeEach(function () {
+            // this is done since cypress session results in 403 sometimes
+            cy.openAuthoring(pagePath);
+        });
+
+        it('insert aem forms Title', function () {
+            dropTitleInSites();
+            cy.deleteComponentByPath(titleDrop);
+        });
+
+        it('check edit dialog availability of Title', function () {
+            testTitleEditDialog(titleEditPathSelector, titleDrop, true);
+        });
+
+    });
+
+    context('Selecting patterns from design policy', function () {
+        const templateDataPath = '/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure',
+            titlePolicy = '[value="' + templateDataPath + '/jcr:content/guideContainer/forms-components-examples/components/form/title' + '"] [data-action="POLICY"]',
+            authoringPagePath = '/content/forms/af/core-components-it/blank',
+            bemDesignDialog = '.cmp-adaptiveform-title__designdialog',
+            titleEditPath = authoringPagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/title",
+            titleDrop = authoringPagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.title.split("/").pop(),
+            titleEditPathSelector = "[data-path='" + titleEditPath + "']";
+            const defaultValue = "H2"
+    
+        beforeEach(function () {
+          cy.openAuthoring(templateDataPath + ".html");
+        });
+    
+        it('Selecting default patterns', function () {
+          cy.get(titlePolicy).click({force: true});
+          cy.get(bemDesignDialog).contains('Title').click();
+          cy.get('[name="./type"]').eq(0).click({ force: true }).then(() => {
+            cy.get('._coral-Menu-item').contains(defaultValue).click({ force: true });
```

Comment:
Please don't make use of private coral selectors in the spec, the spec would break in release/650

Reply:
Removed!